Homework_5/task_1.py

```diff
@@ -1,24 +1,23 @@
 # Напишите программу, удаляющую из текста все слова, в которых присутствуют все буквы "абв"
 
-text = input('Введите текст через пробел и нажмите Enter:\n').split()
-new_text = []
+# Исправлено!
 
-for i in range(len(text)):
-    low = text[i].lower()
-    new_text.append(low)
+text = input('Введите текст через пробел и нажмите Enter:\n').split()
 
 frag_1 = 'а'
 frag_2 = 'б'
 frag_3 = 'в'
 
 text_1 = []
 
-for i in range(len(text)):
-    if frag_1 and frag_2 and frag_3 not in text[i]:
-        a = text[i]
-        text_1.append(a)
+for i in text:
+    if frag_1 in i.lower():
+        if frag_2 in i.lower():
+            if frag_3 in i.lower():
+                continue
+    text_1.append(i)
```

Comment:
Почему же нельзя объединить это в одно условие?
```suggestion
    if not (frag_1 in i.lower() and if frag_2 in i.lower() and if frag_3 in i.lower()):
        text_1.append(i)
```
вот и все